src/network/networkctl.c

```diff
@@ -2145,14 +2188,15 @@ static int parse_argv(int argc, char *argv[]) {
 
 static int networkctl_main(int argc, char *argv[]) {
         static const Verb verbs[] = {
-                { "list",        VERB_ANY, VERB_ANY, VERB_DEFAULT, list_links          },
-                { "status",      VERB_ANY, VERB_ANY, 0,            link_status         },
-                { "lldp",        VERB_ANY, VERB_ANY, 0,            link_lldp_status    },
-                { "label",       VERB_ANY, VERB_ANY, 0,            list_address_labels },
-                { "delete",      2,        VERB_ANY, 0,            link_delete         },
-                { "renew",       2,        VERB_ANY, 0,            link_renew          },
-                { "reconfigure", 2,        VERB_ANY, 0,            verb_reconfigure    },
-                { "reload",      1,        1,        0,            verb_reload         },
+                { "list",            VERB_ANY, VERB_ANY, VERB_DEFAULT, list_links          },
+                { "status",          VERB_ANY, VERB_ANY, 0,            link_status         },
+                { "lldp",            VERB_ANY, VERB_ANY, 0,            link_lldp_status    },
+                { "label",           VERB_ANY, VERB_ANY, 0,            list_address_labels },
+                { "delete",          2,        VERB_ANY, 0,            link_delete         },
+                { "renew",           2,        VERB_ANY, 0,            link_renew          },
+                { "forcerenew",      2,        VERB_ANY, 0,            link_force_renew    },
+                { "reconfigure",     2,        VERB_ANY, 0,            verb_reconfigure    },
+                { "reload",          1,        1,        0,            verb_reload         },
```

Comment:
Why do you reindent them? It seems not necessary.

Reply:
Ouch. first I named this to dhcp-forcerenew. Then I forgot in reindent.